FlowCrypt/src/test/java/com/flowcrypt/email/security/pgp/PgpMsgTest.kt

```diff
@@ -350,9 +344,186 @@ class PgpMsgTest {
 
       if (actualBlock.type in MsgBlock.Type.signedBlocks) {
         val expectedSignature = expectedBlock["signature"].asString.normalizeEol()
-        val actualSignature = ((actualBlock as SignedBlock).signature ?: "").normalizeEol()
+        val actualSignature = ((actualBlock as SignedMsgBlock).signature ?: "").normalizeEol()
         assertEquals(expectedSignature, actualSignature)
       }
     }
   }
+
+  // -------------------------------------------------------------------------------------------
```

Comment:
Approximately 14 test files (.txt) were added below, but this test is using only a handful of them. Where are they being loaded and used?

Reply:
All these files should be used in the tests here.

Reply:
Looks like `mime-email-*.txt` are not used. Will remove them